cypress/integration/unit_tests/color_function_util_test.js

```diff
@@ -65,7 +65,7 @@ describe('Unit tests for color function utility', () => {
     expect(maxMin.is(':visible'));
     expect(maxInput.val()).to.equal('20');
     expect(minInput.val()).to.equal('1');
-    const wings = getCurrentLayers()[0]['color-function']['columns']['wings'];
+    const wings = getCurrentLayers()[0].colorFunction.columns.wings;
```

Comment:
nit, opt: while wings happens to be a legal suffix here, in general the column can be any string. So could keep this ['wings'] to indicate that. Up to you.